packages/core/framework/package.json

```diff
@@ -18,7 +18,12 @@
     "./subscribers": "./dist/subscribers/index.js",
     "./workflows": "./dist/workflows/index.js",
     "./links": "./dist/links/index.js",
-    "./jobs": "./dist/jobs/index.js"
+    "./jobs": "./dist/jobs/index.js",
+    "./utils": "./dist/utils/index.js",
+    "./types": "./dist/types/index.js",
```

Comment:
thought(non-blocking): Think we need to split up the types in the near future. I reckon it will feel weird to consume `@medusajs/framework/types` for many of our types, especially the HTTP ones

Reply:
yes agree, thats another task :) but currently you can import `import { HttpTypes } from '@medusajs/framework/types'`

Reply:
> but currently you can import import { HttpTypes } from '@medusajs/framework/types'

Yeah, this is exactly what I think will feel off.